src/js/view/stage.js

```diff
@@ -1212,3 +1220,44 @@ silex.view.Stage.prototype.moveElements = function(elements, offsetX, offsetY) {
 silex.view.Stage.prototype.isMobileMode = function() {
   return goog.dom.classlist.contains(document.body, 'mobile-mode');
 };
+
+silex.view.Stage.prototype.handleRightClick = function(x, y) {
+
+  if(document.getElementById( 'powDiv' )!=null){
+    var el = document.getElementById( 'powDiv' );
+    el.parentNode.removeChild( el );  
+  }
+
+  var powDiv=goog.dom.createElement('div');
+  powDiv.id='powDiv';
+  
+  var leftPowDiv=goog.dom.createElement('div');
+  leftPowDiv.id='leftPowDiv';
+
+  var imgLeftPowDiv=goog.dom.createElement('img');
+  imgLeftPowDiv.id='imgLeftPowDiv';
+  imgLeftPowDiv.setAttribute('src', '../assets/logo-silex.png');
+
+
+  var rightPowA=goog.dom.createElement('a');
+  rightPowA.id='rightPowA';
+  rightPowA.setAttribute('href', 'http://www.silex.me');
+  rightPowA.setAttribute('target', '_blank');
+  rightPowA.innerHTML='Powered by Silex';
+
+  var myContextHeight=goog.dom.getElementByClass(silex.view.ContextMenu.CLASS_NAME).offsetHeight;
+  var myMenuWidth=goog.dom.getElementByClass('menu-container').offsetWidth;
+
+  powDiv.style.left = (x+myMenuWidth)+'px';
+  powDiv.style.top = (y+myContextHeight)+'px';
+  
+  document.body.appendChild(powDiv);
+  document.getElementById('powDiv').appendChild(leftPowDiv);
+  document.getElementById('leftPowDiv').appendChild(imgLeftPowDiv);
+  document.getElementById('powDiv').appendChild(rightPowA);
+
+  setTimeout(function(){ 
+    var el = document.getElementById( 'powDiv' );
+    el.parentNode.removeChild( el );
```

Comment:
you may want to check that `el.parentNode` is defined, since it won't if the user right clicks many times?